docs/deploying/azure.md

```diff
@@ -0,0 +1,81 @@
+---
+permalink: /docs/deploying/azure/index.html
+layout: docs
+---
+
+If you've been following along with [Getting Started](../README.md), it's time to deploy so you can use it beyond just your local machine.
+[Azure](http://http://azure.microsoft.com/) is a somewhat easy and probably unsupported way to deploy hubot.
+
+You will need to install the azure-cli via npm after you have follow the initial instructions for your hubot.
+
+    % npm install -g azure-cli
+
+Inside your new hubot directory, make sure you've created a git repository, and that your work is committed:
+
+    % git init
+    % git add .
+    % git commit -m "Initial commit"
+
+Then create a GitHub repository for your hubot. This is where zzure will pull your code from instead of needing to deploy directly from your dev machine to Azure.
+
+    % git remote add origin _your GitHub repo_
+	% git push -u origin master
+	
+Create a linked Azure website. In Azure, create a website and select integrated source control. When it asks "where is your source control" select GitHub and link this website to your git repo that you created in the previous step. If you have downloaded the Azure PowerShell modules, you can also do this via PowerShell.
```

Comment:
If there are any Azure documentation to link to, that'd be useful here, particularly for creating a linked Azure website.